### ISSUE TITLE
Invertebrate Data for Ingestion to FBIS

### ISSUE THREAD

**Issue by helendallas** [kartoza/django-bims#1937]
@dimasciput 
The invertebrate data is ready. I don't know how to create a UUID. Please list the steps and I will do, or if Aneri knows how to do this, we can ask her to do it.  

@Aneri16 

Link:
https://www.dropbox.com/s/qy6l6ebvfmuyrn5/InvertebrateConsolidated%202020_03_02.xlsx?dl=0

**Comment by Aneri16** [comment]
@helendallas 
I think I can do it, just need to confirm with Jem about something

**Comment by helendallas** [comment]
@dimasciput 
Okay, UUID created for Invertebrate data:

File is: InvertebrateConsolidated 2020_03_02.xlsx
In folder:
https://www.dropbox.com/sh/u302jtojrjbrlj3/AABJHOtx_lsZ1H4yukzfzoAca?dl=0

@dimasciput Question  -How can we be sure hat the UUIDs generated for inverts and unique from UUIDs generated for algae and fish?

**Comment by dimasciput** [comment]
@helendallas we can't, but the possibility of duplicated uuid is very slim, I can show an error message if there is duplicated UUID and we don't want to update existing data.

**Comment by helendallas** [comment]
@dimasciput Please confirm any issues with invert data ingestion. There were 3 types of errors:

"446 : Bad UUID format" - corrected
"1497 : ValueError : Incorrect Date format", - corrected

Collector/owner and Author(s) fixed

New file name is: SA Invertebrate Data for FBISv3_24 Mar 2020.csv
Link:
https://www.dropbox.com/sh/u302jtojrjbrlj3/AABJHOtx_lsZ1H4yukzfzoAca?dl=0

Please confirm that there are no more issues in preparation for ingestion to the live site

**Comment by helendallas** [comment]
@dimasciput Invertebrate data ready for ingestion. Same file as above

**Comment by helendallas** [comment]
@dimasciput @timlinux 
In checking ingested invertebrate data I came across duplicate records (real or perhaps because data notes or biotopes were incorrectly captured by FRC staffers). As a result I have had to remove two studies from the invertebrate data file for Aneri and Janette to check. And I have manually checked a whole lot of others. 

As a result the revised data file does not have your changes in which unfortunately you didn't keep a record of.  I have change Corydalinae to Corydalidae and revised the Institution header.

If you are able to run a compare against my original file and your ingested file and provide me with your changes then I will carry them over to the new data file. 

The new datafile is in the same dropbox folder, new name
SA Invertebrate Data for FBISv3_16 April 2020.csv
